core/input/input.h

```diff
@@ -219,6 +219,14 @@ class Input : public Object {
 
 	Vector<JoyDeviceMapping> map_db;
 
+	struct JoyAxisButtonMapping {
+		bool replace_axis_as_button = false;
+		float deadzone = 0.2;
```

Comment:
Is the default ever used? If not, it can be `0.0`. No need for magic numbers.

Reply:
The value was inspired by #80299.
`0.0` means that the button is always pressed, which I didn't consider a good default value.
You are right and it would probably be better to provide a default value in `register_axis_as_button`.